packages/flutter/lib/src/physics/spring_simulation.dart

```diff
@@ -124,27 +124,6 @@ class SpringSimulation extends Simulation {
   String toString() => '${objectRuntimeType(this, 'SpringSimulation')}(end: ${_endPosition.toStringAsFixed(1)}, $type)';
 }
 
-/// A [SpringSimulation] where the value of [x] is guaranteed to have exactly the
```

Comment:
Why is this being moved out of the physics library? This would be breaking for anyone that imports it through this library.